### ISSUE TITLE
dup2 failed on arm64

### ISSUE THREAD

**Issue by chenrui333** [bugsnag/panicwrap#18]
```
# github.com/bugsnag/panicwrap
/Users/brew/Library/Caches/Homebrew/go_mod_cache/pkg/mod/github.com/bugsnag/panicwrap@v1.3.1/monitor.go:55:8: undefined: dup2
```

Consider replacing `dup2` for arm64 build

relates to https://github.com/Homebrew/homebrew-core/pull/70684

**Comment by xander-jones** [comment]
👋. hey @chenrui333, build support for `dup2` will be available for `darwin/arm64` in Go 1.16, which should be getting released imminently. Once this new version is released, anyone affected by this should be able to build without error.

**Comment by xander-jones** [comment]
Hey @chenrui333, [Go 1.16](https://tip.golang.org/doc/go1.16) was released today! Are you still seeing this issue if you use this newer version?

**Comment by chenrui333** [comment]
go 1.16 is still [in the pipeline](https://github.com/Homebrew/homebrew-core/pull/71289), maybe I can give a try after it got merged?

**Comment by xander-jones** [comment]
Hey @chenrui333, looks like this got merged down – are you still seeing the same issue with `dup2`?

**Comment by chenrui333** [comment]
@xander-jones I think it is still an issue, see https://github.com/Homebrew/homebrew-core/pull/72439

**Comment by robbydyer** [comment]
For go1.16, removing `!arm64` from https://github.com/bugsnag/panicwrap/blob/master/dup2.go#L1 fixes this issue.

**Comment by johnkiely1** [comment]
This has now been addressed in [v1.3.2](https://github.com/bugsnag/panicwrap/releases/tag/v1.3.2)
Thanks for raising this.
